### PR TITLE
Feature/144/integration of assigning candidate to recruter

### PR DESCRIPTION
**Pull request by Nicolai-boiko**
изображение
изображение

### CODE REVIEW

File: src/app/components/candidate-table/candidate-table/candidate-table.component.ts [hunk 1]
```diff
@@ -77,6 +86,40 @@ export class CandidateTableComponent implements OnInit, AfterViewInit {
...
+  }
+
+  appointCandidateToRecruiter(): void {
+    if (this.recruterId !== undefined) {
+      this.http
+        .post(
+          `${String(environment.API_URL)}/api/recruiters/${this.recruterId}/candidates`,
+          this.candidatesId,
+        )
+        .subscribe();
```
Comment (rastorc3v) [inline src/app/components/candidate-table/candidate-table/candidate-table.component.ts:119]: Add toastr maybe

File: src/styles.scss [hunk 2]
```diff
@@ -44,6 +44,17 @@ body {
...
+  &:disabled,
+  &[disabled]{
+    &:hover {
+      box-shadow: 2px 2px 4px 0 $shadow-color;
+      transform: none;
+      cursor: default;
+    }
+  border: 1px solid #999999;
+  background-color: #cccccc;
+  color: #666666;
```
Comment (vstrabko) [inline src/styles.scss:56]: Move this colors to file with colors.
Other than that looks good

File: src/app/models/candidate.model.ts [hunk 3]
```diff
@@ -1,24 +1,61 @@
 import { ResourceModel } from './resource.model';
+import { Sandbox } from './sandbox.model';
+import { IdName } from './id-name.model';
+
+interface CandidateLanguages {
```
Comment (rastorc3v) [inline src/app/models/candidate.model.ts:5]: move those interfaces to separete folder

**Review by vstrabko — APPROVED** [review]

File: src/app/services/auth.service.ts [hunk 4]
```diff
@@ -84,6 +84,10 @@ export class AuthService {
       );
   }
 
+  userId(): string | undefined {
```
Comment (rastorc3v) [inline src/app/services/auth.service.ts:87]: return string | null (null if currentUser or currentUser?._id is undefined) please

**Review by rastorc3v — CHANGES_REQUESTED** [review]

**Review by rastorc3v — APPROVED** [review]

**Review by ozhurova — APPROVED** [review]

**Pull request merged**